packages/suite-desktop/scripts/build.js

```diff
@@ -58,7 +58,7 @@ console.log(`[Electron Build] Using mocks: ${useMocks}`);
 // must be builded and not included in electron node_modules, because they are in TS.
 // Normal src/ folder is fine, because it's builded by webpack.
 const dependencies = Object.keys(pkg.dependencies).filter(
-    name => !(name.startsWith('@suite-common/') || name === '@trezor/utils'),
+    name => !name.startsWith('@suite-common/'),
```

Comment:
If you will add `@trezor/urls` here, you can remove `build:lib` step from `@trezor/urls` which is better because we want to build as less as possible

Reply:
Maybe would be even better to create some array called `buildedDependencies` and use something like `buildedDependencies.includes(name)` condition, because there could be probably more of them in future

Reply:
9b9534a7d6bfd1560458d177582ef7374fbc4ac5